tests/fixtures/typescript/test-app/tests/rendering/template-only-test.ts

```diff
@@ -3,12 +3,12 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
 
-module('Rendering | template-only', function(hooks) {
+module('Rendering | template-only', function (hooks) {
```

Comment:
Uh, having used JS for what? 25 years?, I will need to get used to this style change... 😅